client/src/plus/collections/collection-list-item.tsx

```diff
@@ -6,6 +6,7 @@ import { Breadcrumbs } from "../../ui/molecules/breadcrumbs";
 import { DropdownMenu, DropdownMenuWrapper } from "../../ui/molecules/dropdown";
 import { docCategory } from "../../utils";
 import { _getIconLabel } from "../common";
+import dayjs from "dayjs";
```

Comment:
```js
import relativeTime from "dayjs/plugin/relativeTime";
dayjs.extend(relativeTime);
```

I still haven't figured out how to do this properly. If it doesn't hurt I'd just add this to be save, otherwise calling `.fromNow()` relies on this being set somewhere else :/